command_line/export_best.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import absolute_import, division, print_function
+
+import logging
+import sys
+
+from libtbx.phil import parse
+from libtbx import Auto
+
+logger = logging.getLogger("dials.command_line.export_best")
+
+help_message = """
+This program is used to export the results of dials processing for the strategy
+program BEST.
+
+"""
+
+phil_scope = parse(
+    """
+
+  intensity = *auto profile sum scale
+    .type = choice(multi=True)
+    .help = "Choice of which intensities to export. Allowed combinations:
+            scale, profile, sum, profile+sum, sum+profile+scale. Auto will
+            default to scale or profile+sum depending on if the data are scaled."
+
+  debug = False
+    .type = bool
+    .help = "Output additional debugging information"
+
+  best {
```

Comment:
We probably don't need the `best` scope if it is now a separate program - i.e. move the parameters in this scope to the top level scope.

Reply:
Done - moved to top level scope and made the change in `xia2` to reflect this.